src/webgpu/api/validation/capability_checks/features/pipeline_statistics_query.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = '';
+
+import { params, pbool, poptions } from '../../../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { ValidationTest } from '../../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('create_query_set')
+  .desc(
+    `
+Tests that create query set with the type of pipeline-statistics shouldn't be valid without pipeline
+statistics query enabled.
+- create query set with the type { occlusion, pipeline-statistics, timestamp }
+- x = pipeline statistics query { enable, disable }
+  `
+  )
+  .params(
+    params()
+      .combine(poptions('type', ['occlusion', 'pipeline-statistics', 'timestamp'] as const))
+      .combine(pbool('pipelineStatisticsQueryEnable'))
+      .expand(p =>
+        poptions('_pipelineStatistics', [
```

Comment:
This can be a local `const pipelineStatistics` inside the fn, doesn't need to be a private parameter.